app/RouteHandlers/AddDonationHandler.php

```diff
@@ -145,7 +145,7 @@ private function getBankDataFromRequest( Request $request ): BankData {
 			->setBankCode( $request->get( 'blz', '' ) )
 			->setBankName( $request->get( 'bankname', '' ) );
 
-		if ( $bankData->hasIban() && !$bankData->hasCompleteLegacyBankData() ) {
+		if ( $bankData->hasIban() && !$bankData->hasBic() && !$bankData->hasCompleteLegacyBankData() ) {
```

Comment:
This logic is misplaced... just saying, don't think this should be fixed in this PR